apps/mosaico/src/Analytics/Analytics.purs

```diff
@@ -0,0 +1,67 @@
+module Mosaico.Analytics where
+
+import Prelude
+
+import Effect (Effect)
+import Effect.Uncurried (EffectFn1, runEffectFn1)
+import Data.Maybe (Maybe(..), fromMaybe)
+import Lettera.Models (Author, Tag)
+import Data.DateTime (DateTime)
+import KSF.User.Cusno (toString, Cusno)
+import KSF.Api.Subscription (Subscription)
+import Data.Foldable (foldMap)
+import Data.Formatter.DateTime (format)
+import KSF.Helpers (dateTimeFormatter)
+import Data.Array (intercalate)
+
+
+type ArticleMetadata = 
```

Comment:
Do we need this intermediate type? Could we convert `Article` directly into `StringArticleMetadata`?